### PR TITLE
Bug 23 carousel arrows

### PR DESCRIPTION
**Pull request by melonholy**

### CODE REVIEW

File: src/containers/couresCarousel/index.js [hunk 1]
```diff
@@ -59,7 +59,7 @@ class CoursesCarousel extends Component {
           className={styles.navButton}
           type="button"
           onClick={this.prevItem}>
-          <IoIosArrowBack color="white" size="50px" />
+          <IoIosArrowBack color="white" className={styles.carouselArrows} />
         </button>
         <div className={styles.carousel}>
           <AliceCarousel
```
Comment (OlegPanasyuk) [inline src/containers/couresCarousel/index.js:64]: Maybe `ref={el => this.Carousel = el}` needs to be replased by `ref={this.CarouselRef}`?

React.CreateRef() is intended for such goals like animation.

File: src/containers/couresCarousel/index.js [hunk 2]
```diff
@@ -76,7 +76,7 @@ class CoursesCarousel extends Component {
           className={styles.navButton}
           type="button"
           onClick={this.nextItem}>
-          <IoIosArrowForward color="white" size="50px" />
+          <IoIosArrowForward color="white" className={styles.carouselArrows} />
         </button>
       </section>
     );
```
Comment (OlegPanasyuk) [inline src/containers/couresCarousel/index.js:80]: So, remove the unnecessary code bonded with the Redux.
```
const mapStateToProps = state => ({});

const mapDispatchToProps = {};

export default connect(
  mapStateToProps,
  mapDispatchToProps
)(CoursesCarousel);
```

File: src/containers/couresCarousel/index.js [hunk 3]
```diff
@@ -20,7 +20,7 @@ const responsive = {
   730: {
     items: 2,
   },
-  350: {
+  370: {
```
Comment (OlegPanasyuk) [inline src/containers/couresCarousel/index.js:22]: Look, we have such endpoints for CSS.
```
$width-xs: 320px;
$width-sm: 576px;
$width-md: 768px;
$width-lg: 992px;
$width-xl: 1200px;
```

Maybe it useful to have such endpoints into the carousel.

File: src/validation/auth.js [hunk 4]
```diff
@@ -42,6 +42,7 @@ export const loginValidate = (email, password) => {
 
   const emailValidate = Joi.validate(email, emailSchema);
   const passwordValidate = Joi.validate(password, passwordSchema);
+  console.log(passwordValidate);
```
Comment (OlegPanasyuk) [inline src/validation/auth.js]: Remove Console executing

**Review by OlegPanasyuk — APPROVED** [review]

File: src/containers/couresCarousel/styles.module.scss [hunk 5]
```diff
@@ -21,10 +28,18 @@
     overflow: hidden;
     outline: none;
     height: 100%;
+    
+    @media screen and (max-width: $width-md) {
+      display: none;
+    }
   }
```
Comment (slavasgod) [inline src/containers/couresCarousel/styles.module.scss:35]: You hide arrows on mobile version, so now it is hard to understand, that this is a carousel, it looks like a single image.

**Review by slavasgod — CHANGES_REQUESTED** [review]

**Review by slavasgod — APPROVED** [review]

**Pull request merged**